### PR TITLE
fix: nameet login validation

### PR DESCRIPTION
**Pull request by nameetjambla**
login frontend validation and GitHub issue number 659

### CODE REVIEW

File: src/app/entry/login/login.component.html [hunk 1]
```diff
@@ -40,7 +51,15 @@ <h2 id="loginTitle" class="text-uppercase font-weight-bold mb-3 mt-2">Login</h2>
                     (click)="toggleFieldTextType()"></i>
                   </span>
                </div>
-
+               <div *ngIf="password1.invalid && (password1.dirty || password1.touched)" class="text-danger w100">
+                <div *ngIf="password1.errors.required" >
+                  Password is required.
+                </div>
+                <div *ngIf="password1.errors.pattern">
+                  Password consist of atleast 8 digits and must contains uppercase,lowercase and special characters for period(@,#,$,'.',&) only.
```
Comment (kunalfleethawks) [inline src/app/entry/login/login.component.html]: Change this to - The length of Password must be 8 characters and must contain combination of letters, numbers and special characters(@,#,$,'.',&).

File: src/app/entry/login/login.component.html [hunk 2]
```diff
@@ -18,7 +18,17 @@ <h2 id="loginTitle" class="text-uppercase font-weight-bold mb-3 mt-2">Login</h2>
...
               </div>
               <input name="username" type="text" placeholder="Email ID or Username"
-                class="form-control login form-control-lg" [(ngModel)]="userName" />
+                class="form-control login form-control-lg" [(ngModel)]="userName" #login="ngModel" required pattern="^[a-zA-Z0-9\.]{6,20}$"/>
+                <div *ngIf="login.invalid && (login.dirty || login.touched)" class="text-danger w100">
+                  <div *ngIf="login.errors.required" >
+                    UserName is required.
+                  </div>
+                  <div *ngIf="login.errors.pattern">
+                    Username consist of atleast 6 digits and must contain  small letters,numbers and special
```
Comment (kunalfleethawks) [inline src/app/entry/login/login.component.html]: Change this to- 
The length of  Username must be 6 characters and may contain combination of letters, numbers and special character(.).

**Review by kunalfleethawks — CHANGES_REQUESTED** [review]
Please address review comment

File: src/app/entry/login/login.component.html (hunk 2 above)
Comment (nameetjambla) [reply src/app/entry/login/login.component.html]: done

**Review by kunalfleethawks — APPROVED** [review]

**Pull request merged**